kibana-reports/server/routes/utils/reportHelper.ts

```diff
@@ -59,7 +62,24 @@ export const createVisualReport = async (
   });
   const page = await browser.newPage();
   await page.setDefaultNavigationTimeout(0);
+  await page.setDefaultTimeout(60000); // 60 sec timeout
+  logger.info(`original query_url ${queryUrl}`);
   await page.goto(queryUrl, { waitUntil: 'networkidle0' });
+  logger.info(`page url ${page.url()}`);
+  logger.info(`page url includes login? ${page.url().includes('login')}`);
+  await delay(5000);
+
+  if (page.url().includes('login')) {
```

Comment:
Can you add some comments about what this section is doing? Looks like an auto-login for FGAC domains?

Reply:
will do